versions/fabric-forge-legacy.txt

```diff
@@ -0,0 +1,12 @@
+org.objectweb.asm.tree.ClassNode org.spongepowered.asm.lib.tree.ClassNode
```

Comment:
I think that mixin uses its own package in 0.8.5 so where is the need

Reply:
what

i literally remap it because mixin relocates ASM in 0.7.11 and below